src/plugins/dvr_controls/public/index.html

```diff
@@ -1,2 +1,2 @@
 <div class="live-info"><%= live %></div>
-<button type="button" class="live-button"><%= backToLive %></button>
+<button type="button" class="live-button" aria-label="Back to live"><%= backToLive %></button>
```

Comment:
Agree, instead of using a literal string "Back to Live", you should use the value of `backToLive`.